test/tsconfig.json

```diff
@@ -3,6 +3,9 @@
   "compilerOptions": {
     "outDir": "../out/test",
   },
+  "files": [
+    "../node_modules/jest-expect-message/types/index.d.ts"
```

Comment:
what is this for?

Reply:
It's part of the Typescript setup for this library: https://github.com/mattphillips/jest-expect-message#configure-typescript